src/test/java/org/tarantool/jdbc/JdbcResultSetMetaDataIT.java

```diff
@@ -176,4 +180,178 @@ public void testWrongColumnAliases() throws SQLException {
+        );
+        try (
+            Statement statement = connection.createStatement();
+            ResultSet resultSet = statement.executeQuery("SELECT * FROM test")
+        ) {
+            ResultSetMetaData rsMeta = resultSet.getMetaData();
+
+            assertFalse(rsMeta.isCurrency(1));
+            assertFalse(rsMeta.isCurrency(2));
+            assertFalse(rsMeta.isCurrency(3));
+            assertFalse(rsMeta.isCurrency(4));
+        }
+    }
+
+    @Test
+    @DisplayName("returned signed columns")
+    public void testSignedColumns() throws SQLException {
+        testHelper.executeSql(
+            "CREATE TABLE test(id INT PRIMARY KEY, double_val DOUBLE, real_val REAL, float_val FLOAT)"
+        );
+        try (
+            Statement statement = connection.createStatement();
+            ResultSet resultSet = statement.executeQuery("SELECT * FROM test")
+        ) {
+            ResultSetMetaData rsMeta = resultSet.getMetaData();
+
+            assertTrue(rsMeta.isSigned(1));
+            assertTrue(rsMeta.isSigned(2));
+            assertTrue(rsMeta.isSigned(3));
+            assertTrue(rsMeta.isSigned(4));
+        }
+    }
+
+    @Test
+    @DisplayName("returned not signed columns")
+    public void testNotSignedColumns() throws SQLException {
+        testHelper.executeSql(
+            "CREATE TABLE test(id INT PRIMARY KEY, char_val VARCHAR(100), text_val TEXT, bin_val SCALAR)"
+        );
+        try (
+            Statement statement = connection.createStatement();
+            ResultSet resultSet = statement.executeQuery("SELECT * FROM test")
+        ) {
+            ResultSetMetaData rsMeta = resultSet.getMetaData();
+
+            assertTrue(rsMeta.isSigned(1));
+            assertFalse(rsMeta.isSigned(2));
+            assertFalse(rsMeta.isSigned(3));
+            assertFalse(rsMeta.isSigned(4));
+        }
+    }
+
+    @Test
+    @DisplayName("returned numeric column types")
+    public void testColumnsNumericTypes() throws SQLException {
+        testHelper.executeSql(
+            "CREATE TABLE test(id INT PRIMARY KEY, f_val FLOAT, d_val DOUBLE, r_val REAL)"
+        );
+        try (
+            Statement statement = connection.createStatement();
+            ResultSet resultSet = statement.executeQuery("SELECT * FROM test")
+        ) {
+            ResultSetMetaData rsMeta = resultSet.getMetaData();
+
+            assertEquals(Types.INTEGER, rsMeta.getColumnType(1));
+            assertEquals(TarantoolSqlType.INTEGER.getTypeName(), rsMeta.getColumnTypeName(1));
+            assertEquals(JdbcType.INTEGER.getJavaType().getName(), rsMeta.getColumnClassName(1));
+
+            // we cannot distinguish numeric types because Tarantool
+            // receives double noSQL type for all the numeric SQL types
+            assertEquals(Types.DOUBLE, rsMeta.getColumnType(2));
+            assertEquals(TarantoolSqlType.DOUBLE.getTypeName(), rsMeta.getColumnTypeName(2));
```

Comment:
Should not those test cases (here and below) being of a kind of black box testing? I mean, check against immediate string constants for type names instead of using TarantoolSqlType / JdbcType.